validator/lib/validateGlb.js

```diff
@@ -11,7 +11,14 @@ module.exports = validateGlb;
 function validateGlb(glb) {
     var version = glb.readUInt32LE(4);
 
+    cp.execFile('./GLTF/exec/gltf_validator', ['./GLTF/glb_sample/Box.glb'], function(err, stdout, stderr){
+        if(err) {console.error(err);}    
+        console.log('stdout', stdout);
+        console.log('stderr', stderr);
```

Comment:
Once promises are set up, this should not call `console.log`, instead the promise should be resolved with the output message.